library/core/class.request.php

```diff
@@ -514,8 +514,18 @@ protected function _loadEnvironment() {
                 safeParseStr($get, $get, $original);
             }
 
-            if (!empty($_SERVER['X_REWRITE'])) {
+            if (!empty($_SERVER['X_REWRITE']) || !empty($_SERVER['REDIRECT_X_REWRITE'])) {
                 $path = $_SERVER['PATH_INFO'];
```

Comment:
I acknowledge this isn't caused by this PR, *BUT*...I recommend this be changed to something like
```php
$path = $this->getValueFrom(self::INPUT_SERVER, 'PATH_INFO')
```
`PATH_INFO` isn't guaranteed to be set.  A notice is thrown from here under my GoDaddy account, because of an invalid array index.